account/src/main/java/com/user/account/controller/AllControlers.java

```diff
@@ -28,38 +49,56 @@ public String home() {
 		// Updating accountBalance after debit
 		
 		
-		@PostMapping("/debit/{accountNumber}/{amount}")
-		public User debit(@PathVariable String accountNumber, @PathVariable String amount) {
+		@PostMapping("/debit/{accountNumber}")
```

Comment:
Why are you taking amount in the body in plain format?

Shouldn't you take it in some sort of class?

What if we need to pass more information like `transaction-method`/`transaction-date`?

Where will we put that information?